src/app/components/plugin-sidebar/plugin-sidebar.component.ts

```diff
@@ -66,7 +66,8 @@ export class PluginSidebarComponent implements OnInit, OnDestroy {
             // and external default templates should not be used
             return this.templateId ?? ALL_PLUGINS_TEMPLATE_ID;
         }
-        return this.templateId;
+        const defaultTemplateId = this.defaultTemplate?.self?.resourceKey?.uiTemplateId ?? null
```

Comment:
This will cause the template id to be written to the url upon navigation, which will prevent changes to the default template to apply. This is not the correct fix to this problem (and I need to replicate the problem myself to see where the issue is)